packages/code-du-travail-frontend/src/common/A11yLink.tsx

```diff
@@ -0,0 +1,30 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+type Props = {
+  children: string;
+} & React.DetailedHTMLProps<
+  React.AnchorHTMLAttributes<HTMLAnchorElement>,
+  HTMLAnchorElement
+>;
+
+export const A11yLink = React.forwardRef<HTMLAnchorElement, Props>(
```

Comment:
I'm not sure the `forwardRef` is used in the code. I didn't move it to a classic React component but I think we can do. What do you think ?

Reply:
it's needed for internal link that are wrapped inside a `<Link>` tag

Reply:
Are you sure? I didn't find any references in the code when searching `A11yLink`.